chart/templates/_helpers.yaml

```diff
@@ -171,7 +171,7 @@ If release name contains chart name it will be used as a full name.
 - name: {{ .Values.dags.gitSync.containerName }}{{ if .is_init }}-init{{ end }}
   image: {{ template "git_sync_image" . }}
   imagePullPolicy: {{ .Values.images.gitSync.pullPolicy }}
-  securityContext: {{- include "localSecurityContext" .Values.dags.gitSync | nindent 4 }}
+  securityContext: {{ include "localPodSecurityContext" .Values.dags.gitSync | nindent 4 }}
```

Comment:
```suggestion
  securityContext: {{ include "containerSecurityContext" (list . .Values.dags.gitSync) | nindent 4 }}
```

Other changes required :point_down: 

### values.yaml
```yaml
dags:
  # [...]
  gitSync:
    # [...]
    securityContexts:
      container: {}
```

and the equivalent schema in 
### values.schemas.json
```json
        "dags": {
            "description": "DAGs settings.",
            "type": "object",
            "x-docsSection": "Airflow",
            "additionalProperties": false,
            "properties": {
                [...]
                "gitSync": {
                    "description": "Git sync settings.",
                    "type": "object",
                    "additionalProperties": false,
                    "properties": {
                        [...]
                        "securityContexts": {
                            "description": "Security context definition for the git sync sidecar. If not set, the values from global `securityContexts` will be used.",
                            "type": "object",
                            "x-docsSection": "Kubernetes",
                            "properties": {
                                "container": {
                                    "description": "Container security context definition for the git sync sidecar.",
                                    "type": "object",
                                    "$ref": "#/definitions/io.k8s.api.core.v1.SecurityContext",
                                    "default": {},
                                    "x-docsSection": "Kubernetes",
                                    "examples": [
                                        {
                                            "allowPrivilegeEscalation": false,
                                            "capabilities": {
                                                "drop": [
                                                    "ALL"
                                                ]
                                            }
                                        }
                                    ]
                                }
                            }
                        },
```